src/NServiceBus.Persistence.CosmosDB/InstallerFeature.cs

```diff
@@ -0,0 +1,39 @@
+﻿namespace NServiceBus.Persistence.CosmosDB
+{
+    using System;
+    using Features;
+
+    class InstallerFeature : Feature
+    {
+        public InstallerFeature()
+        {
+            Defaults(s => s.SetDefault(new InstallerSettings()));
+            DependsOn<SynchronizedStorage>();
+        }
+
+        protected override void Setup(FeatureConfigurationContext context)
+        {
+            var settings = context.Settings.Get<InstallerSettings>();
+            if (settings.Disabled)
+            {
+                return;
+            }
+
+            var databaseName = context.Settings.Get<string>(SettingsKeys.DatabaseName);
```

Comment:
We're assuming these are registered by the customers or defaults are in place. Wonder if we should note it here.

Reply:
We can safely assume because we have set the defaults : https://github.com/Particular/NServiceBus.Persistence.CosmosDB/pull/17/files#diff-b9f4fad9cc365bc37ab878cb08ec4248R17